docs/guides/react-server-cli.md

```diff
@@ -1,113 +1,198 @@
-
-```javascript
-module.exports = {
-	// these will be applied to every page in the site.
-	middleware: ["./FooMiddleware", "./BarMiddleware"],
-
-	// this maps URLs to modules that export a Page class.
-	routes: {
-		BazRoute: {
-			path: ["/"],
-			method: "get", // optional
-			page: "./BazPage"
-		},
-		BakRoute: {
-			path: ["/bak"],
-			page: "./BakPage"
-		}
-	}
-};
+```bash
+$ npm install -g react-server-cli
+$ npm init
+$ react-server init
+$ react-server add-page '/' Homepage
+$ react-server start
 ```
 
 ## What It Does
 
-The CLI builds and runs a `react-server` project, using Express. It compiles JS(X) and CSS into efficiently loadable bundles with code splitting using webpack, and it supports hot reloading of React components on the client-side during development.
+The CLI builds and runs a React Server project, using Express. It compiles
+JS(X) and CSS into efficiently loadable bundles with code splitting using
+webpack, and it supports hot reloading of React components on the client-side
+during development.
 
 ## Built-in Features
 
 ### Babel Compilation
-It's rare to see a project these days in the JavaScript world that isn't at least experimenting with ES2015 and ES7. To make this easier, all code in your project will be run through Babel, and source maps will be generated back to the original file.
+It's rare to see a project these days in the JavaScript world that isn't at
+least experimenting with ES2015 and ES7. To make this easier, all code in your
+project will be run through Babel, and source maps will be generated back to
+the original file.
 
-To take advantage of the Babel compilation, you need to install the Babel plugins and presets you want and reference them in a `.babelrc` file in your code directory. For more on the `.babelrc` format, see [its documentation here](https://babeljs.io/docs/usage/babelrc/).
+To take advantage of the Babel compilation, you need to install the Babel
+plugins and presets you want and reference them in a `.babelrc` file in your
+code directory. For more on the `.babelrc` format, see [its documentation
+here](https://babeljs.io/docs/usage/babelrc/).
 
-## Options
-Smart defaults are the goal, and `react-server-cli` has two base modes: **development** and **production**. `react-server-cli` will determine which base mode it's in by looking at the NODE_ENV environment variable. If it's not "production", then `react-server-cli` will assume we are in development mode.
+## Configuration
 
-### Ways to add options
+### Routes
+
+This is where URLs are mapped to pages.  It's also where global middleware
+(applied to all pages in the site) is defined.
+
+By default this is created at the site's root directory as `routes.json`.
 
-There are three ways to pass options to the CLI, through the command line, `.reactserverrc` JSON files, or as a `reactServer` entry in `package.json` files. It searches for options this way:
+```json
```

Comment:
I feel like the routes.json format should be stored in the docs, and this should be a link to those docs instead of a code snippet